<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sdl_qt_rgbClass</class>
 <widget class="QWidget" name="sdl_qt_rgbClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sdl_qt_rgb</string>
  </property>
  <widget class="QLabel" name="video1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(85, 255, 255)</string>
   </property>
   <property name="text">
    <string>video</string>
   </property>
  </widget>
  <widget class="QLabel" name="video2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Open1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pix1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YUV420P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGRA</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="width1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="height1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="set_fps1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>500</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="set_fps2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>500</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QComboBox" name="pix2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YUV420P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGRA</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="width2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>440</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Open2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>390</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="height2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLabel" name="fps1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fps2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>50</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 85, 0);</string>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sdl_qt_rgb.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Open1</sender>
   <signal>clicked()</signal>
   <receiver>sdl_qt_rgbClass</receiver>
   <slot>Open1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>255</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Open2</sender>
   <signal>clicked()</signal>
   <receiver>sdl_qt_rgbClass</receiver>
   <slot>Open2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open1()</slot>
  <slot>Open2()</slot>
 </slots>
</ui>
